pajbot/apiwrappers/twitch/helix.py

```diff
@@ -726,7 +726,30 @@ def fetch_all_moderators(self, broadcaster_id, authorization):
         """Calls the _fetch_moderators_page function using the broadcaster_id parameter.
         broadcaster_id is a required field and must match the user ID in authorization."""
         moderator_ids = self._fetch_all_pages(self._fetch_moderators_page, broadcaster_id, authorization)
```

Comment:
For style reasons I‘d like to have an empty line between each method please, I‘m honestly surprised that `black` isn‘t complaining about this